### DISCUSSION TITLE
Não são as cartas…

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#475] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/nao-sao-as-cartas/

**Comment by maryon-telvanni** [comment]
![null](<https://i.imgur.com/aXus3Gc.png>)

**Comment by one-eyed-willy** [comment]
Nunca que eu iria adivinhar que esse meme é de um filme de 1972. Pra mim era o Zach Galifianakis :rofl:
